otlp/traces.go

```diff
@@ -125,6 +120,21 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					}
 					if isError {
 						attrs["error"] = true
+
+						// Copy over span event error attributes because:
+						// 1. They are common and high-value for error investigations
+						// 2. It sucks to have to look at span events in our trace UI today to hunt these down on an error span
+						// 3. This makes bubble up better because you can see these error details without having to query the span events
+						for _, seventAttr := range sevent.Attributes {
+							if seventAttr.Key == "exception.message" ||
```

Comment:
Should we flatten all span events into Events?

Reply:
Please do this with a switch statement instead of the ||.